addons/editor-dark-mode/addon.json

```
@@ -1373,6 +1373,31 @@
         "stageHeader": "#e8edf1",
         "border": "#00000026"
       }
+    },
+          {
+      "name": "ScratchLab",
+      "id": "scratch-lab",
+      "description": "The colors used by the ScratchLab's project editor.",
```

It's officially called "Scratch Lab" with a space:
```suggestion
      "name": "Scratch Lab",
      "id": "scratch-lab",
      "description": "The colors used by the Scratch Lab's project editor.",
```

---

Don't know how I keep on forgetting the space...

---

You could be thinking about ScratchX which didn't have a space (I used to mess up Scratch Labs' name before realizing because of that)